ts/features/idpay/onboarding/xstate/services.ts

```diff
@@ -93,12 +107,13 @@ const createServicesImplementation = (
     const dataPromise: Promise<undefined> = pipe(
       response,
       E.fold(
-        _ => Promise.reject(OnboardingFailureType.GENERIC),
+        _ => Promise.reject(OnboardingFailureEnum.GENERIC),
         _ => {
-          if (_.status !== 204) {
```

Comment:
This is a general question. What do you think about using an explicit name instead _ (only for right case)?

Reply:
Having an explicit name can make the code easier to read. In this case, I went with the `_` to keep it consistent with the rest of the code. Plus, the pipe right there makes it pretty clear what we're dealing with, so I don't think it'll be a problem. But yes, overall, I think that using explicit names are a better approach.

Reply:
Yes it's right. I agree with you, it's not a problem. Anyway the point for me is how to make the code readable to a dev (like a new entry) not yet familiar with fp-ts and pipes.

Reply:
Yeah, in that case newcomers can have some issue understanding the code. I'll quickly make a commit to make things more clear!